addons/mail/views/ir_actions_server_views.xml

```diff
@@ -34,6 +34,37 @@
                     </group>
                     <field name="activity_note" class="oe-bordered-editor" placeholder="Add a description to your activity..." invisible="state != 'next_activity'"/>
                 </xpath>
+                <xpath expr="//group[@name='action_content']" position="inside">
+                    <field name="discuss_template_id"
+                           placeholder="Choose a template..."
+                           invisible="state != 'discuss_post'"
+                           required="state == 'discuss_post'"
+                           context="{'default_model': model_name, 'no_create': False}"/>
```

Comment:
Consistency: use standard tabs instead of alignment, and tag close on a next line, with the same indent as the opening of the tag.